chemprop/cli/train.py

```diff
@@ -1028,3 +1058,61 @@ def main(args):
     logging.basicConfig(stream=sys.stdout, level=logging.DEBUG, force=True)
     args = parser.parse_args()
     TrainSubcommand.func(args)
+
+
+def summarize(task_type: str, dataset: MoleculeDataset) -> tuple[list, list]:
+    if task_type in ["regression", "regression-mve", "regression-evidential"]:
+        y = np.array([datapoint.y[0] for datapoint in dataset.data])
+        y_mean = y.mean()
+        y_std = y.std()
+        y_median = np.median(y)
+        mean_dev_abs = np.abs(y - y_mean)
+        num_1_sigma = (mean_dev_abs < y_std).sum()
+        num_2_sigma = (mean_dev_abs < 2 * y_std).sum()
+        frac_1_sigma = num_1_sigma / len(y)
+        frac_2_sigma = num_2_sigma / len(y)
```

Comment:
```suggestion
        frac_1_sigma = (mean_dev_abs < y_std).sum() / len(y)
        frac_2_sigma = (mean_dev_abs < 2 * y_std).sum() / len(y)
```
Stylistic change that you can ignore if you'd like